projects/packages/videopress/src/client/block-editor/hooks/use-video-data/index.ts

```diff
@@ -25,31 +30,62 @@ export default function useVideoData( id: VideoId ): UseVideoDataProps {
 		 * Fetches the video videoData from the API.
 		 */
 		async function fetchVideoItem() {
-			try {
-				const response: WPV2mediaGetEndpointResponseProps = await apiFetch( {
-					path: `/wp/v2/media/${ id }`,
-				} );
-
-				setIsRequestingVideoData( false );
-				if ( ! response?.jetpack_videopress ) {
-					return;
+			if ( guid ) {
+				try {
+					const response: WPCOMRestAPIVideosGetEndpointResponseProps = await apiFetch( {
+						url: `https://public-api.wordpress.com/rest/v1.1/videos/${ guid }`,
```

Comment:
Should we have a constant for this endpoint?

Reply:
Yes, let's figure out where we can organize the code. Ideally, I think we could use the `state/` folder it's consumed also by VP dashboard. We need to be careful sharing that code.